site/docs/v1/tech/apis/scim/SCIMEnterpriseUser.adoc

```diff
@@ -0,0 +1,159 @@
+[.endpoint]
+.URI
+--
+[method]#DELETE# [uri]#/api/scim/resources/v2/EnterpriseUsers`\{userId\}`#
+--
+==== Request Parameters
+
+[.api]
+[field]#userId# [type]#[UUID]# [optional]#Optional#::
+The FusionAuth userId.
+
+:http_method: DELETE
+
+=== Response
+
+This API does not return a JSON response body.
+
+The DELETE endpoint will return a 204 status code upon success or one of the standard error status codes.
+
+:webhook_event:
+:create_user:
+include::/docs/v1/tech/apis/_standard-post-response-codes.adoc[]
+:create_user!:
+:webhook_event!:
+
+For FusionAuth SCIM endpoints, any error responses will be returned in standard SCIM schema. See more details in the link:/docs/v1/tech/apis/scim[SCIM API Overview].
+
+== Retrieve a EnterpriseUser
+
+This API is used to retrieve a FusionAuth User in SCIM schema format through a SCIM request.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Retrieves a EnterpriseUser through a SCIM request
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/scim/resources/v2/EnterpriseUsers/`\{userId\}`#
+--
+==== Request Parameters
+
+[.api]
+[field]#userId# [type]#[UUID]# [optional]#Optional#::
+The FusionAuth userId.
+
+:http_method: GET
+
+=== Response
+
+The response for this API contains the User in SCIM schema format.
+
+:webhook_event:
+:create_user:
+include::/docs/v1/tech/apis/_standard-post-response-codes.adoc[]
+:create_user!:
+:webhook_event!:
+
+For FusionAuth SCIM endpoints, any error responses will be returned in standard SCIM schema. See more details in the link:/docs/v1/tech/apis/scim[SCIM API Overview].
+
+:create_user:
+include::/docs/v1/tech/apis/scim/_scim-enterpriseuser-response-body.adoc[]
+:create_user!:
+
+== Update a EnterpriseUser
+
+This API is used to update a new FusionAuth User from a SCIM request. This is a true update meaning that the FusionAuth User will be overwritten with only the data contained in the request. It is NOT a partial update.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Updates a EnterpriseUser from a SCIM request
+[.endpoint]
+.URI
+--
+[method]#PUT# [uri]#/api/scim/resources/v2/EnterpriseUsers/`\{userId\}`#
+--
+==== Request Parameters
+
+[.api]
+[field]#userId# [type]#[UUID]# [optional]#Optional#::
+The FusionAuth userId.
+
+:http_method: PUT
+include::/docs/v1/tech/apis/scim/_scim-enterpriseuser-request-body.adoc[]
+
+=== Response
+
+The response for this API contains the User that was just updated in SCIM schema format.
+
+:webhook_event:
+:create_user:
+include::/docs/v1/tech/apis/_standard-post-response-codes.adoc[]
+:create_user!:
+:webhook_event!:
+
+For FusionAuth SCIM endpoints, any error responses will be returned in standard SCIM schema. See more details in the link:/docs/v1/tech/apis/scim[SCIM API Overview].
```

Comment:
```suggestion
For SCIM endpoints, error responses will be returned in standard SCIM schema. See more details in the link:/docs/v1/tech/apis/scim[SCIM API Overview].
```

Reply:
I don't know that there is value in saying `FusionAuth SCIM` instead of SCIM. Are there other kinds of SCIM endpoints we'll be documenting?